pkg/gui/services/custom_commands/session_state_loader.go

```diff
@@ -181,19 +182,30 @@ type SessionState struct {
 }
 
 func (self *SessionStateLoader) call() *SessionState {
+	var commit = commitShimFromModelCommit(self.c.Contexts().LocalCommits.GetSelected())
+
+	if commit == nil {
+		commit = commitShimFromModelCommit(self.c.Contexts().ReflogCommits.GetSelected())
+
+		if commit == nil {
+			commit = commitShimFromModelCommit(self.c.Contexts().SubCommits.GetSelected())
+		}
+	}
```

Comment:
This logic doesn't make sense: we need to test which panel is focused, and set commit to LocalCommits.GetSelected() if it's the local commits panel, or to ReflogCommits.GetSelected() if it's the reflogs panel, or to SubCommits.GetSelected() if it's the subcommits panel. Or to nil if it's not any of these, although we might consider setting it to LocalCommits.GetSelected() in that case because that's probably a useful default.

I think you can use `self.c.CurrentSideContext()` to find out which panel is focused.